dashcam/package/object-detection/files/detect_hdc.py

```diff
@@ -79,7 +79,14 @@ def combine_images(images, grid_size, model_size):
 
     return combined_img, orig_images
 
-def transform_box(box, model_size, grid_size, index):
+CLASS_NAMES = ['face', 'person', 'license-plate', 'car', 'bus', 'truck', 'motorcycle', 'bicycle']
+
+def get_label(class_id):
+    if 0 <= class_id < len(CLASS_NAMES):
+        return CLASS_NAMES[class_id]
+    return None
+
+def transform_box(box, model_size, grid_size, index, scale_config, class_id):
```

Comment:
Is this code identical to detect_hdc.py in odc-api? https://github.com/Hivemapper/odc-api/pull/228/files

If so would it be possible to copy this from odc-api instead of duplicating the changes here?